src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const partnerMessage = person.isMarried ? (
+    `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
Create a variable for `person.sex === 'm' ? 'wife' : 'husband'`
Don't use ternary inside another ternary